src/AaveV3Update_20231002matic/AaveV3Update_20231002matic.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Gauntlet Recommendations to Lower stMATIC/MaticX non-emode LT/LTV on Polygon v3
+author: Paul Lei, Watson Fu, Nick Del Zingaro, Walter Li, Jonathan Reem, Nick Cannon, Sarah Chen, Dana Tung
+discussions: https://governance.aave.com/t/arfc-gauntlet-recommendation-to-lower-stmatic-maticx-non-emode-lt/14859
+---
+
+## Summary
+
+Gauntlet recommends lowering stMATIC, MaticX non-emode LT/LTV on Polygon v3 to improve the risk profile associated with MATIC LSTs.
+
+Reducing MATIC LST non-emode LT to lower borrowing power can help combat the following.
+
+- Risks of reduced WMATIC supply in the future, which may cause long-term growth risk for Polygon v3
+- Risks of reduced capacity for recursive LST - WMATIC borrowing
+- Risks associated with increased stablecoin borrowing against LST collateral, amidst decreasing MATIC LST liquidity
+
+
+## Specification
+
+
+| Chain | Asset | Action | Current Value | New Value
+|-------|-------|-----| ---| --- |
+| Polygon | stMATIC | Lower non-emode LT| 65% | 60%
+| Polygon | stMATIC | Lower non-emode LTV| 50%| 45%
+| Polygon | MaticX | Lower non-emode LT| 67%| 62%
+| Polygon | MaticX | Lower non-emode LTV| 58%| 45%
+
+
+
+## Implementation
+
+The proposal implements changes on Polygon v3 using the following pre-deployed payloads:
+  - [Polygon](https://github.com/GauntletNetworks/aave-proposals/blob/628e17e84c9d6b8177adaad22aa87a69a045e829/src/AaveV3Update_20231002matic/AaveV3Polygon_20231002matic.sol)
```

Comment:
if you want our ci to pin this, i think you need to use bgd-labs/aave-proposals/blob/main, but i guess you already pinned manually, so doesn't matter